acceptance/tests/peering/peering_connect_namespaces_test.go

```diff
@@ -266,61 +285,40 @@ func TestPeering_ConnectNamespaces(t *testing.T) {
 				})
 			}
 
-			logger.Log(t, "checking that connection is successful")
-			if cfg.EnableTransparentProxy {
-				k8s.CheckStaticServerConnectionSuccessful(t, staticClientOpts, staticClientName, fmt.Sprintf("http://static-server.virtual.%s.%s.consul", c.destinationNamespace, staticServerPeer))
-			} else {
-				k8s.CheckStaticServerConnectionSuccessful(t, staticClientOpts, staticClientName, "http://localhost:1234")
-			}
+			if c.ACLsAndAutoEncryptEnabled {
+				logger.Log(t, "checking that the connection is not successful because there's no allow intention")
+				if cfg.EnableTransparentProxy {
+					k8s.CheckStaticServerConnectionMultipleFailureMessages(t, staticClientOpts, staticClientName, false, []string{"curl: (56) Recv failure: Connection reset by peer", "curl: (52) Empty reply from server", "curl: (7) Failed to connect to static-server.ns1 port 80: Connection refused"}, "", fmt.Sprintf("http://static-server.virtual.%s.%s.consul", c.destinationNamespace, staticServerPeer))
```

Comment:
`"curl: (7) Failed to connect to static-server.ns1 port 80: Connection refused"` 

Will it always be `ns1` even though you have a testcase above that uses `default` as the namespace?

Reply:
It will always deploy it to the kubernetes namespace `ns1` but will be registered in the `deafult` consul namespace or another consul namespace depending on the testcase.

Reply:
Actually, that is a good catch. we have this assertion the has been copied around for a while that needed updating!